.circleci/config.yml

```diff
@@ -44,17 +44,13 @@ jobs:
       - save_cache:
           name: Saving cache (frontend)
           paths:
-            - src/client/node_modules
-          key: v1-dependencies-{{ checksum "src/client/package.json" }}
+            - src/server/frontEnd/node_modules
+          key: v1-dependencies-{{ checksum "src/server/frontEnd/package.json" }}
 
       - run:
-          name: Static Analysis (backend)
+          name: Static Analysis (backend & frontend)
```

Comment:
You should use always the same name to describe frontend. FrontEnd or frontend choose wisely.